File: Patterns/Manuscript/manuscript/code/java/DesignPatterns/src/main/java/bridge/Television.java
package bridge;

class Television implements Device {
    private boolean isOn;
    private int volume;
    private int channel;

    public Television() {
        this.isOn = false;
        this.volume = 50; // Default volume
        this.channel = 1; // Default channel
    }

    public void powerOn() {
        isOn = true;
        System.out.println("Television powered on");
    }

    public void powerOff() {
        isOn = false;
        System.out.println("Television powered off");
    }

    public void adjustVolume(int volume) {
        this.volume = volume;
        System.out.println("Adjusting television volume to " + volume);
    }

    public void changeChannel(int channel) {
        this.channel = channel;
        System.out.println("Changing television channel to " + channel);
    }
}

File: code/csharp/DesignPatterns/bridge/Television.cs
namespace Bridge;
using System;

public class Television : IDevice
{
    private bool _isOn;
    private int _volume;
    private int _channel;

    public Television()
    {
        this._isOn = false;
        this._volume = 50; // Default volume
        this._channel = 1; // Default channel
    }

    public void PowerOn()
    {
        _isOn = true;
        Console.WriteLine("Television powered on");
    }

    public void PowerOff()
    {
        _isOn = false;
        Console.WriteLine("Television powered off");
    }

    public void AdjustVolume(int volume)
    {
        this._volume = volume;
        Console.WriteLine("Adjusting television volume to " + volume);
    }

    public void ChangeChannel(int channel)
    {
        this._channel = channel;
        Console.WriteLine("Changing television channel to " + channel);
    }
}
